src/renderer/services/wallet/util.test.ts

```diff
@@ -173,4 +174,32 @@ describe('services/wallet/util/', () => {
       ])
     })
   })
+
+  describe('getBalanceByAsset', () => {
+    it('get balance by asset', () => {
+      expect(
+        getBalanceByAsset(AssetBNB)([
+          {
+            asset: ASSETS_TESTNET.TOMO
+          },
+          {
+            asset: ASSETS_TESTNET.BOLT
+          },
+          {
+            asset: ASSETS_TESTNET.FTM
+          },
+          {
+            asset: ASSETS_TESTNET.BNB
+          },
+          {
+            asset: AssetRuneNative
+          }
+        ] as WalletBalances)
```

Comment:
As discussed in DM: 
- Don't use `as` for type casting, it's just like cheating the type system. 
- Use `eqOAsset` to test result in a very safety way

Reply:
@veado IMHO its ok to use typecasting in tests. sometimes we could need only 1 or 2 values for unit testing but target interface might be really big. and if something changes what does not affect tests we will have tests failed and need to fix it. anyway if there is something missed we will find it out when tests fail

Reply:
@thatStrangeGuyThorchain Don't agree. It's just the lack of TypeScript you can do it like that. And it's the knowledge you might have about internals of a function like `getBalanceByAsset`. However, if something of its internal will be changed, this way won't make sure, that tests will fail then. Create mock factories if you scare about building "bigger" objects for tests. Or change types of function parameters and return types to be more generic. But that's not what we do need in our code in this case, where we have to make sure we will get a `WalletBalance` (not parts) from it.

Reply:
yeah, make sense